packages/web-forms/src/components/ControlHint.vue

```diff
@@ -0,0 +1,19 @@
+<script setup lang="ts">
+import type { AnyControlNode as QuestionNode } from '@getodk/xforms-engine';
+defineProps<{question: QuestionNode}>();
+</script>
+
+<template>
+	<div v-if="question.currentState.hint" class="hint">
+		{{ question.currentState.hint?.asString }}
+	</div>
+</template>
+
+<style scoped lang="scss">
+
+.hint {
+	font-size: 0.85rem;
+	color: var(--surface-600);
```

Comment:
My understanding is that surface colors are different based on dark/light theme. 
`--surface-0` is white in light theme and black in dark theme.

See following files:
primevue-sass-theme/themes/material/material-light/_variables.scss
primevue-sass-theme/themes/material/material-dark/_variables.scss

But this is no longer true in PrimeVue 4 auro preset. There are several places where I have used `surface` colors, I will change that to use semantic tokens in PrimeVue upgrade PR.

Reply:
My understanding, which may be wrong, is that the `surface` variables are intended to color surfaces, e.g. their backgrounds. Am I misunderstanding that? Or are you saying that you'd prefer to keep consistency of usage, in spite of its inconsistency of design/intent?

Reply:
Happened to see this fly by and I have noticed that PrimeVue in its own examples uses surface tokens in places I wouldn't expect including lots of `color` settings. You can see some of that in their PrimeVue 4 examples too: https://primevue.org/theming/styled/#presets 

I don't know if they have a reasoning for it but I agree that trying to use `surface` tokens only for backgrounds/fills would be ideal. I think we will eventually want to allow users to set colorful surfaces but we would not want that to apply to any text. Being consistent about using `text-*` tokens for text seems it will help us in the future.

Reply:
got it, let me use `--text-color-secondary` here.